tests/main/dirs-not-shared-with-host/task.yaml

```diff
@@ -8,7 +8,7 @@ details: |
   consistent behaviour across various distributions.
 
 environment:
-    DIRECTORY/ssl: /etc/ssl
+    DIRECTORY/apparmord: /etc/apparmor.d
```

Comment:
heh, nice typo

Reply:
You mean `apparmord` (without the `.`) ? Or did I make a typo somewhere else? Fwiw, the "appamord" is needed because spread does not like a "." in the variants.

Reply:
Sorry. I was confused by the diff initially, and then forgot to update it before posting the review.

/etc/apparmor.d does not exist in all distros we run this test on, how about /etc/dbus-1? I checked that it's present on centos 7/8 and amazon linux, the rest should have it too.

Reply:
Yeah, /etc/apparmor.d does not exist on all distro - so does /etc/ssl that we used before. I'm happy to do a followup cleaning up this test. We need to pick something from `*dirs_from_core` (mount-support.c) and AFAICT all dirs will not exist everywhere. We could pick /etc/nsswitch.conf which should be universal but it's a file not a dir so the test would be slightly changed.

Reply:
Is /etc/alternatives in centos-*, amzn2, opensuse-*, fedora-* and should be in ubuntu/debian. It's only missing on arch, which is fine to blacklist for the test, wdyt?

Reply:
Thank you!

Reply:
@bboozzoo do we want to make the change to /etc/alternatives to cover more distros?

Reply:
Unfortunately /etc/alternatives is present in most distros (excl. arch), but there is no counterpart directory in the core* snaps, so the test would be ineffective again. Staying with /etc/apparmor.d and blacklisting systems where it is not present is as good as it gets for now.